sql/mints.ts

```diff
@@ -0,0 +1,65 @@
+export const mintsTableSql = `
```

Comment:
This is fine for now, but I would recommend looking into [Prisma](https://www.prisma.io/) so that we can generate migrations and interactions via code 👍